src/Hpack/Render.hs

```diff
@@ -351,7 +351,6 @@ data RenderBuildTool = BuildTools String | BuildToolDepends String
 
 renderBuildTool :: (BuildTool,  DependencyVersion) -> RenderBuildTool
 renderBuildTool (buildTool, renderVersion -> version) = case buildTool of
```

Comment:
I assume it is a stylistic choice, but I noted the continued use of `case` even though type `BuildTool` has only one constructor.